docs/pages/desktop-access/getting-started.mdx

```diff
@@ -196,6 +212,45 @@ Computer Configuration > Policies > Windows Settings > Security Settings > Publi
   ![Import Teleport CA](../../img/desktop-access/ca.png)
 </Figure>
 
+### Publish the Teleport CA to the Active Directory domain
+
+<Admonition type="note" title="Differences when using AWS Managed Active Directory">
+  When using AWS Managed Active Directory, you should run this command using an account which is part of the
+  `AWS Delegated Domain Administrators` group, such as the AWS-provided `admin` account.
+</Admonition>
+
+On a machine which is joined to your domain and logged in as an account in the `Domain Administrators` group,
+run the two commands below at a PowerShell prompt to publish the Teleport CA to your Active Directory
+domain (using the path to the exported Teleport `user-ca.cer` file that you copied above):
+
+```powershell
+certutil –dspublish –f <PathToCertFile.cer> RootCA
+```
+
+This step enables the domain controllers to trust the Teleport CA, which will allow smart card logons via Teleport to succeed.
+
+### Publish the Teleport CA to the NTAuth Store
+
+In order for authentication with Teleport-issued certificates to succeed, the
+Teleport CA needs to be published to the enterprise NTAuth store. Teleport will
+periodically publish its CA after it is able to authenticate, but this step
+needs to be performed manually the first time in order for Teleport to have LDAP
```

Comment:
This step is _required_ for Teleport 9+ because we use x509 certs to auth for LDAP.

It is not required for Teleport 8 because we use an LDAP password.

Reply:
I can remove it from this PR.